Utilities/DataFlow/CMakeLists.txt

```diff
@@ -94,5 +97,8 @@ O2_GENERATE_TESTS(
 
 O2_GENERATE_MAN(NAME TimeframeReaderDevice)
 O2_GENERATE_MAN(NAME TimeframeWriterDevice)
+O2_GENERATE_MAN(NAME SubframeBuilderDevice)
 
 target_compile_options(test_TimeframeParser PUBLIC -O0 -g)
```

Comment:
The canonical solution is to make one more PR build for Debug and then one gets both optimised and non optimised builds. The solution now is an awkward mixture.